bundles/org.openhab.binding.deconz/README.md

```diff
@@ -235,6 +239,8 @@ Bridge deconz:deconz:homeserver [ host="192.168.0.10", apikey="ABCDEFGHIJ" ] {
 Switch                  Livingroom_Presence     "Presence Livingroom [%s]"          <motion>        { channel="deconz:presencesensor:homeserver:livingroom-presence:presence" }
 Number:Temperature      Livingroom_Temperature  "Temperature Livingroom [%.1f °C]"  <temperature>   { channel="deconz:temperaturesensor:homeserver:livingroom-temperature:temperature" }
 Number:Dimensionless    Livingroom_Humidity     "Humidity Livingroom [%.1f %%]"     <humidity>      { channel="deconz:humiditysensor:homeserver:livingroom-humidity:humidity" }
+String                  LIVING_ROOM_VOC_LABEL    "Air quality Livingroom [%s]"      <none>          { channel="deconz:airqualitysensor:homeserver:livingroom-voc:airquality" }
+Number                  LIVING_ROOM_VOC          "Air quality [%d ppb]"             <none>          {           channel="deconz:airqualitysensor:homeserver:livingroom-voc:airqualityppb" }
```

Comment:
Other example items are prefixed "Livingroom", but here casing is "LIVING_ROOM". Suggest to harmonize.

Reply:
Good point I haromnised the doc.